app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        if person["name"] not in Person.people:
+            Person(person["name"], person["age"])
+    for person in people:
+        prsn = Person.people[person["name"]]
+        if "husband" in person and person["husband"] is not None:
+            prsn.husband = find_partner(person["husband"])
+            if prsn.husband is not None:
+                prsn.husband.wife = prsn
```

Comment:
It seems unnecessary to create a separate  `find_partner()` function just to retrieve a value from a list
```suggestion
            if Person.people.get(person["husband"]) is not None:
                prsn.husband.wife = prsn
```

Reply:
Привіт, а треба заново пул реквест робити або цей буде перевірений ?